File: 1346. Check If N and Its Double Exist.php
class Solution {
    function checkIfExist($arr) {
         $n = count($arr);
        for ($i = 0; $i < $n; $i++) {
            for ($j = $i + 1; $j < $n; $j++) {
                if ($arr[$i] * 2 == $arr[$j] || $arr[$j] * 2 == $arr[$i]) {return true;}
            }
        }
        return false; 
    }
}

File: 1346. Check If N and Its Double Exist.rs
impl Solution {
    pub fn check_if_exist(arr: Vec<i32>) -> bool {
        let n = arr.len();
        for i in (0..n) {
            for j in (i+1..n) {
                if arr[i] * 2 == arr[j] || arr[j] * 2 == arr[i] {return true;}
            }
        }
        return false; 
    }
}
